extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -103,7 +103,7 @@ amp-carousel .amp-carousel-button.amp-disabled {
 }
 
 .-amp-slides-container.-amp-no-scroll {
-  overflow-x: hidden !important;
+  -webkit-overflow-scrolling: auto !important;
```

Comment:
Not needed, only needed where there is momentum scrolling. Have tested with firefox and is fine.